app/src/main/res/values-night/colors_dark.xml

```diff
@@ -27,4 +27,6 @@
     <item name="colorChatInputButton" type="color">@color/button_background_dark</item>
     <item name="colorChatHedvigBackground" type="color">@color/button_background_dark</item>
 
+    <item name="colorOfferSignButton" type="color">@color/violet_300</item>
```

Comment:
I assume we're removing this once we've switched out all buttons to be this color on dark theme? 🙂

Reply:
yes :)